core/src/main/scala/sgl/geometry/Rect.scala

```diff
@@ -1,7 +1,7 @@
 package sgl.geometry
 
 /** an AABB Rect. */
-class Rect(var left: Float, var top: Float, var width: Float, var height: Float) {
```

Comment:
There's some code in the platformer example that is using mutation of Rect. I don't know if it's a good idea, but it was done as a way to limit garbage collection when I was profiling the example.

Reply:
I guess you already fixed that in a follow-up commit. It's just a FYI then, I'm not very used to the github code review system :)